library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -139,6 +152,36 @@ def self.create(snapshot_type, description, previous = nil)
     end
     private_class_method :create
 
+    # detects if module runs in initial stage before scr is switched to target system
```

Comment:
np: It would be nice adding yard-style method documentation (`@return` and that kind of things).